worker/src/main/scala/org/apache/celeborn/service/deploy/worker/FetchHandler.scala

```diff
@@ -355,7 +355,11 @@ class FetchHandler(
       callback: RpcResponseCallback): Unit = {
     checkAuth(client, Utils.splitShuffleKey(shuffleKey)._1)
     workerSource.recordAppActiveConnection(client, shuffleKey)
-    workerSource.startTimer(WorkerSource.OPEN_STREAM_TIME, shuffleKey)
+    val requestId = Utils.makeOpenStreamRequestId(
+      shuffleKey,
+      client.getChannel.id().toString,
```

Comment:
nit: 
```
client.getChannel.id().asLongText()
```

Reply:
`client.getChannel.id().toString` equals `client.getChannel.id().asShortText()` seems this value is nearly unique enough for metrics, which is sufficient.And It would better use  `client.getChannel.id().asShortText()` to replace `client.getChannel.id().toString` @turboFei  WDYT?

Reply:
> client.getChannel.id().toString equals client.getChannel.id().asShortText()
It sounds good, thanks